### ISSUE TITLE
performance of test result

### ISSUE THREAD

**Issue by JongMokKim** [NickLucche/lane-detection#4]
Hello! Thank you for great work.

I'm interested in training the network with my custom data.
To validate your training script, I ran train.py with not changing train parameters (only data path)
What I'm concerning is that the performance is much more poor than you reported:
acc : 92.02, F1 : 56.95, Prec:43.99, Recall : 87.72
(your report : acc :95.5, F1:65, Prec:55, Recall:81)

which parameter should I tune? or is there any other factor can affect the performance?

thanks in advance!

**Comment by NickLucche** [comment]
Sorry for the delayed answer, I might have submitted the wrong model, let me look into that and I'll get back to you.
In the meantime, you should try running the `test.py` script instead, making sure to test on the provided test set.
